language/move-lang/functional-tests/tests/libra/natives/event_no_resource_handle.move

```diff
@@ -0,0 +1,11 @@
+script {
+use 0x1::Event;
+use 0x1::LBR::LBR;
+use 0x1::Libra;
+
+fun main(account: &signer) {
+    let handle = Event::new_event_handle<Libra::Libra<LBR>>(account);
+    Event::destroy_handle<Libra::Libra<LBR>>(handle);
+}
+}
+// check: MoveSourceCompilerError resource type not copyable
```

Comment:
If we leave in this test, it needs to be an exp file. Or it needs to be moved out of functional tests

Reply:
Similarly, this is covered in `constraints_not_satisfied_lvalues_bind_type.move`.